pronotepy/ent.py

```diff
@@ -474,3 +474,37 @@ def monbureaunumerique(username, password):
     cookies = requests.utils.cookiejar_from_dict(
         requests.utils.dict_from_cookiejar(session.cookies))
     return cookies
+
+def ent_essonne(username, password):
+    """
+    ENT Essonne
+
+    Parameters
+    ----------
+    username : str
+        username
+    password : str
+        password
+
+    Returns
+    -------
+    cookies : cookies
+        returns the ent session cookies
+    """
+    # ENT / PRONOTE required URLs
+    ent_login = "https://www.moncollege-ent.essonne.fr/auth/login"
+    # Required Headers
+    headers = {
+        'connection': 'keep-alive',
+        'User-Agent': 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:73.0) Gecko/20100101 Firefox/73.0'}
+
+    payload = {
+        'email' : username,
+        'password' : password,
+    }
+    # ENT Connection
+    session = requests.Session()
+
+    cookies = requests.utils.cookiejar_from_dict(requests.utils.dict_from_cookiejar(session.cookies))
```

Comment:
This is useless since you do not have any request before that line. I would suggest removing it.

Reply:
Ok i can try again with this. It's just that I was inspired by the Ent il de France

Reply:
Yeah but here it's not needed, as you are not carrying cookies over multiple requests that need cookies set by older requests.